src/Compilers/Core/Portable/SourceGeneration/Nodes/NodeStateTable.cs

```diff
@@ -393,7 +401,7 @@ private static (T chosen, EntryState state) GetModifiedItemAndState(T previous,
             }
         }
 
-        private readonly struct TableEntry
+        internal readonly struct TableEntry
```

Comment:
don't love this piece.  any ideas?

Reply:
Maybe make a new type next to it `CachedEntryEnumerable`?